requirements.txt

```diff
@@ -34,3 +34,6 @@ rsa==4.9
 sqlparse==0.5.3
 urllib3==2.3.0
 virtualenv==20.28.1
+langchain
+langchain-groq
+langchain-community
```

Comment:
@yunjin08 kindly specify the version of langchain here. Also, add a newline at the end. You can do something like this if you are using a separate virtual environment:
```
pip freeze > requirements.txt
```